dev-packages/node-integration-tests/suites/public-api/LocalVariables/test.ts

```diff
@@ -16,6 +16,9 @@ const EXPECTED_LOCAL_VARIABLES_EVENT = {
                 arr: [1, '2', null],
                 obj: { name: 'some name', num: 5 },
                 ty: '<Some>',
+                bool: false,
+                num: 0,
```

Comment:
So one thing here, in case of the number, we would previously report `'<0>'` instead of `0`. I think this should be okay but wanted to double check if there are implications to this change. Anyone have an idea?